README.md

```
@@ -56,6 +56,39 @@ If you're a developer who's spotted a bug issue and have a fix, or simply have t
 
 Visit the [Tour Operator Reviews Plugin on Github](https://github.com/lightspeeddevelopment/to-reviews/) and submit a Pull Request with your updates.
 
+## Development & Coding Standards
+
+This plugin follows the [WordPress Coding Standards](https://developer.wordpress.org/coding-standards/) to ensure code quality and consistency:
+
+### PHP Standards
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)